api/database/migration_exif.go

```diff
@@ -51,11 +71,35 @@ func migrateExifFields(db *gorm.DB) error {
 
 		return nil
 	})
+}
 
-	if err != nil {
-		return err
+func processFlash(exifCol gorm.ColumnType, db *gorm.DB) error {
```

Comment:
`normalizeFlash`?

Reply:
regarding the "process" part of the names here and later: what do you think about "convert" or "migrate" alternatives? I'm not a native speaker) but "normalize" doesn't look accurate in this case, as it means (to me) to align something with a "normal" reference value...

Reply:
How about `parse`? I see the function `migrateExifFieldsFlash` converts the flash value from a string to a int, according to [the map](https://github.com/photoview/photoview/pull/1073/files/43199f4bace36bc9eb13de68424bbd5acd61abb1#diff-9b219c58eb0d2e05ccb3c24f2ee2924e670ca7526eabb158521272e2dfe034f0R20).

To me,  `convert` is a general word, no special meaning. `migrate` means converting a valid data from one pattern to another patten. `parse` means converting a general data to a structed data.

Reply:
done